cfgov/jinja2/v1/_includes/organisms/carousel.html

```diff
@@ -50,33 +66,19 @@ <h6 class="o-carousel_thumbnail-text">{{ title }}</h6>
         </button>
 
         <ul class="o-carousel_items">
-
-            {% call render_carousel_item( 'Start small, Save Up campaign' ) %}
-                Lorem ipsum dolor sit amet, ei ius adhuc inani iudico, labitur instructior ex pri. Cu pri inani constituto, cum aeque noster commodo cu.
-            {% endcall %}
-
-            {% call render_carousel_item( 'Financial Literacy Month' ) %}
-                Show belly cat milk copy park pee walk owner escape bored tired cage droppings sick vet vomit favor packaging over toy cats making all the muffins get scared by doggo also cucumerro . Sit in window and stare oooh, a bird, yum meow meow.
-            {% endcall %}
-
-            {% call render_carousel_item( 'This is a key initiative' ) %}
-                Doggo ipsum dat tungg tho wrinkler heck borking doggo aqua doggo, pats heckin borkdrive. Heckin angery woofer fluffer thicc blep, big ol pupper length boy shooberino super chub, heckin shibe. Smol borking doggo with a long snoot for pats boofers borkdrive such treat fat boi, puggorino borkf long woofer.
-            {% endcall %}
-
-            {% call render_carousel_item( 'This is title fourth item' ) %}
-                Robot ipsum Zygolike perfluoroeae benzocy equiphage telluriose quinhood exbiile hysterophyceae tetride. Italocarp nonalyte silicoistically isoaria Panule genitoatory gamolog herpetoonymy. Thanatostat Sinophorous poikiloite toxoid dermatocarpous osmioopsy xennamancy. Omphaloate nanoonymy ekageny euphane forthphyll deuterolepry auxostomy. Choleid symule vicelike italoderm cacory bradynasty speleoite enneanomics. Papaphilous petaworthy eurie contrataxy poikilooideae contraal limnont quinathon umbeont. Dextomy raceade.
-            {% endcall %}
-
+            {% for item in value._items %}
+                {{ render_carousel_item( item ) }}
+            {% endfor %}
         </ul>
 
     </div>
 
     <ul class="o-carousel_thumbnails">
-        {{ render_carousel_thumbnail( 'Start small, Save Up campaign', true ) }}
-        {{ render_carousel_thumbnail( 'Financial Literacy Month' ) }}
-        {{ render_carousel_thumbnail( 'This is a key initiative' ) }}
-        {{ render_carousel_thumbnail( 'This is title fourth item' ) }}
+        {% for item in value._items %}
+            {{ render_carousel_thumbnail( item, loop.first ) }}
+        {% endfor %}
     </ul>
 </div>
-
 {% endmacro %}
+
+{% if value %}{{ render(value) }}{% endif %}
```

Comment:
TRIVIAL: Space inside parens.

Reply:
Whoops, thanks! Shia beat me to it but I'll open another PR.